README.md

```diff
@@ -1,92 +1,224 @@
+X, y = digits.data, digits.target
+
+svm = SVC(kernel='rbf', gamma='scale', C = 0.5).fit(X, y)
+print(svm.score(X, y))
+```
+
+In addition daal4py provides an option to replace some scikit-learn methods by oneDAL solvers which makes it possible to get a performance gain **without any code changes**. This approach is the basis of Intel distribution for python scikit-learn. You can patch stock scikit-learn by using the only following commandline flag
+```bash
+python -m daal4py my_application.py
+```
+Patches can also be enabled programmatically:
+```python
+from sklearn.svm import SVC
+from sklearn.datasets import load_digits
+from time import time
+
+svm_sklearn = SVC(kernel="rbf", gamma="scale", C=0.5)
+
+digits = load_digits()
+X, y = digits.data, digits.target
+
+start = time()
+svm_sklearn = svm_sklearn.fit(X, y)
+end = time()
+print(end - start) # output: 0.141261...
+print(svm_sklearn.score(X, y)) # output: 0.9905397885364496
+
+from daal4py.sklearn import patch_sklearn
+patch_sklearn() # <-- apply patch
+from sklearn.svm import SVC
+
+svm_d4p = SVC(kernel="rbf", gamma="scale", C=0.5)
+
+start = time()
+svm_d4p = svm_d4p.fit(X, y)
+end = time()
+print(end - start) # output: 0.032536...
+print(svm_d4p.score(X, y)) # output: 0.9905397885364496
+```
+
+
+For more details browse our [daal4py documentation](https://intelpython.github.io/daal4py/).
+
+
+## oneDAL Samples
+
+<img align="right" style="display:inline;" height=300 width=550 src="docs/readme-charts/intel%20oneDAL%20Spark%20samples%20vs%20Apache%20Spark%20MLlib.png"></a>
+
+Samples is a non official part of a project, that contains examples of how oneDAL can be used in different applications.
+
+**oneDAL Spark Samples is a good example.**
+oneDAL provides scala / java interfaces that match Apache Spark MlLib API and use oneDAL solvers under the hood. This implementation allows you to get a 3-18X increase in performance compared to default Apache Spark MLlib.
+
+  ## Examples
+
+Except C++ and Python API oneDAL also provide API for C++ SYCL and Java languages. Check out tabs below for more examples.
+- [C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/cpp)
+- [C++ SYCL\*](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/cpp_sycl)
+- [Java](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/java)
+- [Python](https://github.com/IntelPython/daal4py/tree/master/examples)
 
 Data Examples for different computation modes:
 
-- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/batch)
-- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/distributed)
-- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/online)
+- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/data/batch)
+- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/data/distributed)
+- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/data/online)
```

Comment:
Not sure we should reference only DAAL examples

Reply:
It seems to me that keeping data references is not a good idea. We have a description of the computation modes in the paragraphs above, as well as an example of how to run distributed algorithms with daal4py